cite-gradle-plugin/src/main/kotlin/com/jakewharton/cite/plugin/gradle/CitePlugin.kt

```diff
@@ -41,7 +41,7 @@ public class CitePlugin : KotlinCompilerPluginSupportPlugin {
 		}
 		target.afterEvaluate {
 			check(applied) {
-				"The Cite plugin requires either the Kotlin multiplatform or JVM plugin"
+				"The Cite plugin requires either the Kotlin Multiplatform, JVM or Android plugin"
```

Comment:
```suggestion
				"The Cite plugin requires either the Kotlin Multiplatform, JVM, or Android plugin"
```
I'm an oxford comma stickler!